cmake/Modules/FindMxnet.cmake

```diff
@@ -71,6 +77,4 @@ else()
   set(Mxnet_COMPILE_FLAGS "${Mxnet_COMPILE_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
 endif()
 
-mark_as_advanced(Mxnet_INCLUDE_DIRS Mxnet_LIBRARIES Mxnet_COMPILE_FLAGS Mxnet_USE_MKLDNN Mxnet_USE_CUDA Mxnet_VERSION)
-
-mark_as_advanced(Mxnet_INCLUDE_DIRS Mxnet_LIBRARIES Mxnet_COMPILE_FLAGS Mxnet_USE_MKLDNN Mxnet_USE_CUDA Mxnet_VERSION)
+mark_as_advanced(Mxnet_INCLUDE_DIRS Mxnet_LIBRARIES Mxnet_COMPILE_FLAGS Mxnet_USE_MKLDNN Mxnet_USE_ONEDNN Mxnet_USE_CUDA Mxnet_VERSION)
```

Comment:
@leezu you have duplicated this line in https://github.com/horovod/horovod/commit/bec40a7d9ccad52bb0dc580aeac09b2b7427fc2a#diff-bd892e8ebe35ec5332e410bd9d22dd664bfb6d92ab50b7938aa4dcf3e180bd4eR66:

https://github.com/horovod/horovod/blob/0acb1a53cc832c8c580e03c1a29c0e14d95234a1/cmake/Modules/FindMxnet.cmake#L74-L76

Was that intentional?

Reply:
No, I don't think so

Reply:
Thanks, I will remove it then